utils/docker_utils.go

```diff
@@ -0,0 +1,153 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"io"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"strings"
+	"time"
+)
+
+type ContainerConfig struct {
+	Name   string
+	Image  string
+	Mounts []mount.Mount
+}
+
+func NewDockerClient(host string) *client.Client {
+	if err := os.Setenv("DOCKER_HOST", host); err != nil {
+		log.Print(err)
+	}
+	cli, _ := client.NewEnvClient()
+	return cli
+}
+
+func ListContainer(cli *client.Client) {
+	ctns, err := cli.ContainerList(context.Background(), types.ContainerListOptions{All: true})
+	if err != nil {
+		log.Fatal(err)
+	}
+	for _, container := range ctns {
```

Comment:
why are we writing to the stdout? (not sure If understood this correctly, though)

Reply:
I can remove this func actually.. gonna do that